### PR TITLE
gh-config: add page

### PR DESCRIPTION
**Pull request by TheLastGimbus**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Add a new page for `gh config`

I could not figure out what does `--host` option do - neither does it say anywhere in the official documentation. If someone ever knew, it would be nice to fill that

Ref: #4701

### CODE REVIEW

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:1]:
```suggestion
# gh config
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:6]: We try to avoid `you`, `your`, etc.
```suggestion
- Display what Git protocol is being used:
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:14]: Based on `git-config.md`:
```suggestion
- Set text editor to Vim:
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:10]:
```suggestion
- Set protocol to SSH:
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:12]: The argument options should be highlighted:
```suggestion
`gh config set git_protocol {{ssh}}`
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:16]:
```suggestion
`gh config set editor {{vim}}`
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:18]:
```suggestion
- Reset to default text editor:
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:20]:
```suggestion
`gh config set editor {{""}}`
```

File: pages/common/gh-config.md
Comment (navarroaxel) [inline pages/common/gh-config.md:24]:
```suggestion
`gh config set prompt {{disabled}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @TheLastGimbus, I left some comments.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — COMMENTED** [review]
LGTM, but I'm just wondering if it'd be worth adding two examples to the top for generic `get`/`set` to cover any future additional config properties. i.e.

```markdown
- Get a specific configuration value:

`gh config get {{key}}`

- Set a specific configuration value:

`gh config set {{key}} {{value}}`
```

Also, I can't find the `--host` option that was mentioned, so not sure what that is. 🤷🏻

**Comment by TheLastGimbus** [comment]
I added `set` example - `get` is obvious to figure out from it

> Also, I can't find the --host option

When you type `gh config set --help` - `-h, --host string   Set per-host setting` - I tried if it's about repo-specific, but couldn't get it to work - it's probably for different Enterprise hosts :shrug:

**Review by owenvoke — APPROVED** [review]
LGTM! Ah, yeah. I'd assume that's for per-Enterprise settings. 👍🏻 We can probably exclude that though for now.

**Comment by TheLastGimbus** [comment]
Hey, so, can someone merge this? Everything is approved (I don't know why cla doesn't complete, but i've signed :ok_hand: ) soo...

**Review by sbrl — APPROVED** [review]
Thanks, @TheLastGimbus!

**Comment by owenvoke** [comment]
Sorry, I was planning on merging but forgot to after I'd approved it. Thanks @TheLastGimbus 👍🏻

**Pull request merged**
